docs/index.yaml

```diff
@@ -143,10 +125,28 @@ entries:
     urls:
     - https://aerospike.github.io/helm-charts/aerospike-pulsar-outbound-1.0.0.tgz
     version: 1.0.0
+  aerospike-vector-search:
+  - apiVersion: v2
+    appVersion: 0.4.0
+    created: "2024-06-05T15:09:39.393452+03:00"
+    description: A Helm chart for Aerospike Vector Search
+    digest: f66c0163471586d199efa4c88ce73a45978d17071ba8e59ba6ff8fabeebf4876
+    icon: https://avatars0.githubusercontent.com/u/2214313?s=200&v=4
+    keywords:
+    - Aerospike
+    - Vector
+    maintainers:
+    - email: developers@aerospike.com
+      name: Aerospike
+    name: aerospike-vector-search
+    type: application
+    urls:
+    - https://aerospike.github.io/helm-charts/aerospike-vector-search-0.1.0.tgz
+    version: 0.1.0
```

Comment:
Shouldn't this version be 0.2.0?

Reply:
The tag in `aerospike-vector-search-examples` repo is `0.1.0` that's why decided to leave it as it is basically overwriting the existing one

Reply:
Let us go up a version so we do an upgrade. I think we don't need to rely on that tag.

Reply:
@arrowplum No problem will rename the tag on dockerhub to be `0.2.0`